Resources/Help/preferences/prefsui.md

```diff
@@ -14,6 +14,8 @@ Here you can specify if you would like to have a "light" or a "dark" theme on th
 ### Preferred Language
 Select the language in which you would like to use JASP.
 
+If you would like to improve one translation, or to add a new language, please check this document: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-guideline-translators.md
```

Comment:
I think the text used in https://github.com/jasp-stats/jasp-desktop/pull/4625#issuecomment-919948474 works a bit better. Also good to use the jasp-stats.org url here: https://jasp-stats.org/translation-guidelines

Reply:
I tend to copy paste this md into https://jasp-stats.org/translation-guidelines